File: apzkr-pzpi-21-5-ivakin-nikita/Task1-Server/src/main/java/nikita/ivakin/apzpzpi215ivakinnikitatask2/model/dto/SupplyCarDTO.java
package nikita.ivakin.apzpzpi215ivakinnikitatask2.model.dto;

import lombok.Builder;
import lombok.Data;
import nikita.ivakin.apzpzpi215ivakinnikitatask2.model.dto.groups.LogisticCompanyDTO;

@Data
@Builder
public class SupplyCarDTO {

    private Integer id;

    private Integer tier;

    private Integer supplyRequestId;

    private Integer logisticCompanyId;

    private SupplyRequestDTO supplyRequest;

    private LogisticCompanyDTO logisticCompany;
}

File: apzkr-pzpi-21-5-ivakin-nikita/Task3-WebClient/src/app/services/models/supply-car-dto.ts
/* tslint:disable */
/* eslint-disable */
import { LogisticCompanyDto } from '../models/logistic-company-dto';
import { SupplyRequestDto } from '../models/supply-request-dto';
export interface SupplyCarDto {
  id?: number;
  logisticCompany?: LogisticCompanyDto;
  logisticCompanyId?: number;
  supplyRequest?: SupplyRequestDto;
  supplyRequestId?: number;
  tier?: number;
}
